### ISSUE TITLE
Testing with my scanned documents.

### ISSUE THREAD

**Issue by zokai** [DevashishPrasad/CascadeTabNet#90]
I would like to try CascadeTabNet with my own images.
I used main.py and set the followings path:

image_path 
xmlPath 
config_fname 

However I don't know what to set for these variables:
checkpoint_path = "path to checkpoint directory"
epoch = 'epoch_file.name'

Does your github repository come with a pre-trained model based on your dataset?
or I have to train your model locally?

Thanks

**Comment by cabudies** [comment]
Hi Zokai,

Would suggest point checkpoint to "cascade_mask_rcnn_hrnetv2p_w32_20e.py" and epoch to "epoch_36.pth"

**Comment by kaustubh-shirpurkar** [comment]
I agree with cabudies answer and would also recommend you to go through ~[Model Zoo of CascadeTabNet] (https://github.com/DevashishPrasad/CascadeTabNet#6-model-zoo). Model zoo includes different checkpoint files along with dataset on which they were trained. 

 > **epoch_36.pth**

This model  was trained on a highly specific dataset. I would recommend to start with a model trained on more general 
dataset such as 

>**epoch_24.pth**

and try different models whichever suits your need.

**Comment by zokai** [comment]
Thank you.

**Comment by VincentFTS** [comment]
Hi, did you succeed in detecting your table structure with the given checkpoint ?

**Comment by kaustubh-shirpurkar** [comment]
Yes I was able to get table structure out using same checkpoint.

**Comment by VincentFTS** [comment]
Was it a border or a borderless table ?
Did you use Colab ?
Did you tweak anything to get good results ?

**Comment by VincentFTS** [comment]
Could you share your python code ?

**Comment by kaustubh-shirpurkar** [comment]
I used colab version. Here is the link for colab notebook. I think you need to make some changes while using images. Code is pretty self explanatory. https://colab.research.google.com/drive/16GzDZqfWCf3Kt6_EOk7FKZ7sHTYii-_w?usp=sharing
